solutions/bio1/Unlock/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { SCodeBlock, SText, STitle } from "components";
+import { brace } from "solutions/util";
+
+export const Unlock = () => {
+  return (
+    <>
+      <STitle>Observation</STitle>
+      <SText>
+        This problem cannot be solved by bruteforcing every possible combination
+        of button presses, since the number of possible combinations is ${brace("3^{25} \\approx 8.47*10^{11}")}$, which
+        would be far too slow. However, is it possible to bruteforce only some set of the buttons,
+        and then use the results of those button presses to determine the rest of the button presses?
+      </SText>
+
+      <STitle>Solution</STitle>
+      <SText>
+        Just bruteforce all possible combinations of the top 5 buttons, and 
+        then use the results of those button presses to determine the rest of the 
+        button presses. Notice that after the top 5 buttons have been pressed, for
+        each light in the top row, the only button that can affect its state, thus 
+        set it to the correct state, is the button directly below it (for button A, this 
+        is button F, for button B, this is button G, and so on). Thus, we can try to build a 
```

Comment:
I think it doesn't matter in this case, but consider using dollar signs around `$A$`, `$B$` and so on. It might look better, but it might not - up to you.